packages/webview_flutter/tizen/src/ewk_internal_api_binding.cc

```diff
@@ -35,6 +35,8 @@ bool EwkInternalApiBinding::Initialize() {
       dlsym(handle_, "ewk_view_ime_window_set"));
   view.KeyEventsEnabledSet = reinterpret_cast<EwkViewKeyEventsEnabledSetFnPtr>(
       dlsym(handle_, "ewk_view_key_events_enabled_set"));
+  view.SupportVideoHoleSet = reinterpret_cast<EwkViewSupportVideoHoleSetFnPtr>(
+      dlsym(handle_, "ewk_view_set_support_video_hole"));
```

Comment:
Doesn't this require a profile distinction?

Reply:
Although it is not used in `TV_PROFILE`, I thought it was not necessary to distinguish registering in `EwkViewProcTable`. Do you think it is necessary?